source/documentation/breaking-changes/calc-functions.md

```diff
@@ -0,0 +1,121 @@
+instead of functions.
+
+[`round()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/round
+[`abs()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/abs
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+{% codeExample 'calc-function-bad', false %}
+@function rem($number) {
+  @return $number % 10;
+}
+
+/* This example would parse `rem()` as a calc function, not as the declared
+function above. Since `rem()` requires two parameters, this would throw a
+parameter error. */
+.sidebar {
+  margin-left: rem(9) * 1px;
+}
+
+===
+@function rem($number)
+  @return $number % 10
+
+// This example would parse `rem()` as a calc function, not as the declared
+// function above. Since `rem()` requires two parameters, this would throw a
+// parameter error.
+.sidebar 
+  margin-left: rem(9) * 1px
+
+{% endcodeExample %}
+
+However, if the custom `rem()` function was imported to another file, it would
+be parsed as the declared function since it's assigned to a different namespace.
+
+{% codeExample 'calc-function-import', false %}
+<===> foo.scss
+@function rem($number) {
+  @return $number % 10;
+}
+
+<===> main.scss
+// This example would not throw an error.
+@use 'foo';
+
+.sidebar {
+  a: foo.rem(9) * 1px;
+}
+===
+<===> foo.scss
+@function rem($number)
+  @return $number % 10
+
+<===> main.scss
+// This example would not throw an error.
+@use 'foo'
+
+.sidebar
+  a: foo.rem(9) * 1px
+
+{% endcodeExample %}
+
+{% headsUp %}
+  If the module was imported [_without a namespace_] 
+  calc functions would override the custom declared functions.
+  For instance, `@use 'foo' as *` would change the call to `a: rem(10)`,
+  which would result in an error.
+
+[_without a namespace_]: /documentation/at-rules/use/#choosing-a-namespace
+{% endheadsUp %}
+
+To preserve the current behavior, use the [`call()`] and [`get-function()`] methods
+from the `meta` module.
+
+[`get-function()`]:http://localhost:8080/documentation/modules/meta/#get-function
+[`call()`]: /documentation/modules/meta/#call
```

Comment:
1. make sure to update these per the comment above
2. don't use the localhost:8080 url